scripts/ConnectBoxManage.sh

```diff
@@ -455,6 +455,22 @@ function set_staticsite () {
     exit 1;
   fi
 
+  if [ ! -f /media/usb0/index.html ]; then
```

Comment:
This will be created even if the device isn't being put into static site mode (this function is run when the static site state is being toggled). Please move it within the appropriate conditional block (`if [[ "$val" == "true" ]]; then`)